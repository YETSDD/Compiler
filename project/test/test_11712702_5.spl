int test_11712702_5(int b_0, int s)
{
  char b = '\x0aaa';
  char 0O00 = 'a';
  
  return s;
}
