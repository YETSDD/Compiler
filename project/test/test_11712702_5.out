Error type A at Line 3 : Invalid hex character '\x0aaa'
Error type A at Line 4 : Unknown lexeme 0O00
